ansible/pbTestScripts/testJDK.sh

```diff
@@ -1,14 +1,12 @@
 #!/bin/bash
 
-mv /vagrant/pbTestScripts/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* ~
+mv $HOME/openjdk-build/workspace/build/src/build/linux-x86_64-normal-server-release/images/jdk8* $HOME
 export TEST_JDK_HOME=$(find ~ -maxdepth 1 -type d -name "*jdk8u*"|grep -v ".*jre.*")
```

Comment:
You seem to have changed the previous line from `~` to `$HOME` but not this one - any particular reason?
Also is there a reason not to just set `TEST_JDK_HOME` to the original location instead of moving it?

Reply:
Ah, Sorry, missed that.
 As for setting TEST_JDK_HOME, I initially moved it as it was easier to move both the JDK and the JRE, then set just the JDK - see : https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/943#discussion_r326642706